Svc/TraceFileLogger/TraceFileLogger.hpp

```diff
@@ -0,0 +1,196 @@
+// ======================================================================
+// \title  TraceFileLogger.hpp
+// \author sreddy
+// \brief  hpp file for TraceFileLogger component implementation class
+// ======================================================================
+
+#ifndef Svc_TraceFileLogger_HPP
+#define Svc_TraceFileLogger_HPP
+
+#include "Svc/TraceFileLogger/TraceFileLoggerComponentAc.hpp"
+#include "Svc/TraceFileLogger/ArrayProc.hpp"
+#include <Os/File.hpp>
+#include <Fw/Types/Assert.hpp>
+#include <Fw/Buffer/Buffer.hpp>
+
+// some limits.h don't have PATH_MAX
+#ifdef PATH_MAX
+#define FILE_PATH_MAX PATH_MAX
+#else
+#define FILE_PATH_MAX 255
+#endif
+
+// some limits.h don't have NAME_MAX
+#ifdef NAME_MAX
+#define FILE_NAME_MAX NAME_MAX
+#else
+#define FILE_NAME_MAX 255
```

Comment:
delete this and use the macro in Config/FpConfig.h instead for file path and fine name